src/math.ts

```diff
@@ -71,18 +72,34 @@ const adjustXYWithRotation = (
   if (side === "e" || side === "ne" || side === "se") {
     x += deltaX * (1 - cos);
     y += deltaX * -sin;
+    if (isResizeFromCenter) {
+      x += deltaX * cos;
+      y += deltaX * sin;
+    }
```

Comment:
```suggestion
    if (isResizeFromCenter) {
      x += deltaX;
    } else {
      x += deltaX * (1 - cos);
      y += deltaX * -sin;
    }
```
Can we simplify like this?

Reply:
I think in the case for `isResizeFromCenter` we will need to sum deltaX to y: `y += deltaX * sin;`

Reply:
Hm, I just refactored without understanding the calculation.
if `isResizeFromCenter`,
it was:
```js
y += deltaX * -sin;
y += deltaX * sin;
```
which becomes
```js
y += 0;
```
no???

Reply:
Sorry, I was confused with the other sides, you're right for that scenario `e`, `ne` and `se` that works fine. updating 👍